### DISCUSSION TITLE
option for stdout

### DISCUSSION THREAD

**Discussion by lgkahn** [adriankumpf/tesla_auth#45] [Ideas]
any way to generate the token in one file and the refresh token into another so i can put in a web server direcxtory and use it that way instead of having to copy and paste every 8 hours.. thanks

**Comment by lgkahn** [comment]
the token file needs to look something like this we we can just put on a web server and have our apps query it via json.

anyone.. i dont have a compile environment or i would add it.. shouldnt be too hard thanks

C:\utils>cat teslatoken
{"access_token":"qts-13691793bdaa194c4283b9f45210b29d0504457267b48d2c70298e615079a","token_type":"bearer","expires_in":3888000,"refresh_token":"cf134c8de7e8ba9ccf2eb8d88c5b744801064480f10396a952bf0536409cb847","created_at":1625310844}
C:\utils>

**Comment by adriankumpf** [comment]
You usually don't need to get regenerate the tokens every 8 hours. Instead, try refreshing the access token: https://tesla-api.timdorr.com/api-basics/authentication#refreshing-an-access-token
